tests/validation/tests/v1_api/resource/daemonset_1.json

```diff
@@ -0,0 +1,43 @@
+{
+  "type": "apps.daemonset",
+  "metadata": {
+    "namespace": "default",
+    "name": "test-daemonset-1"
+  },
+  "metadata.annotations": {
+  },
+  "metadata.labels": {
+  },
```

Comment:
I'm still unsure of how well these json test data will scale over time. But at the same time there's not much to choose from (besides the client approach).

- yaml
- serialize python code to json at runtime
- write factories and convert to json

Reply:
You are right, we have only those limited options for generating resources, and none of them is easy to maintain at scale, in my opinion. 
Let's leave this conversation open to the team to see if anyone has any insights.